wpimath/src/test/native/cpp/MathUtilTest.cpp

```diff
@@ -65,6 +65,69 @@ TEST(MathUtilTest, ApplyDeadbandLargeMaxMagnitude) {
       frc::ApplyDeadband(100.0, 20.0, std::numeric_limits<double>::infinity()));
 }
 
+TEST(MathUtilTest, ApplyDeadband2dUnityScale) {
+  const Eigen::Vector2d zero(0.0, 0.0);
+
+  EXPECT_EQ((Eigen::Vector2d(0.0, 1.0)),
```

Comment:
The usual style for these is `Eigen::Vector2d{{0.0}, {1.0}}` instead of `Eigen::Vector2d(0.0, 1.0)`. Was it incompatible with the Google Test macros?